index.css

```diff
@@ -0,0 +1,30 @@
+body {
+  font-family: monospace;
+  background-color: lavender;
+  flex-direction: column;
+  align-items: center;
+}
+
+header {
```

Comment:
we can use the `align-items` css property to align the text in the middle via y-axis